### PR TITLE
Cleanup, Package.json, and Removed Graphs

### PR DESCRIPTION
**Pull request by V1rtuousCycle**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -2,11 +2,12 @@ const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
 const db = require('../db/index.js');
-const PORT = process.env.PORT || 3000;
 const session = require('express-session');
 
+const PORT = process.env.PORT || 3000;
```
Comment (awiw642) [inline server/index.js:7]: What has changed with this?

File: package.json [hunk 2]
```diff
@@ -5,44 +5,38 @@
   "engine": "node 6.11.5",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint --fix .",
-    "react-dev": "webpack -d --watch",
-    "server-dev": "nodemon server/index.js",
+    "festart": "webpack -d --watch",
+    "bestart": "nodemon --inspect server/index.js",
```
Comment (awiw642) [inline package.json:9]: What is the reason for the change in the name?

File: package.json [hunk 3]
```diff
@@ -5,44 +5,38 @@
...
     "seed": "node db/seed.js",
-    "start": "node server/index.js",
     "test": "jest",
     "test:watch": "jest --watch",
+    "lint": "./node_modules/.bin/eslint --fix .",
```
Comment (awiw642) [inline package.json:13]: This was deleted above and readded here? Why?

Comment (V1rtuousCycle) [reply package.json:13]: Line change, so that people could see the more used commands first.

File: package.json (hunk 2 above)
Comment (V1rtuousCycle) [reply package.json:9]: its a slightly easier convention to remember, imo.

File: server/index.js (hunk 1 above)
Comment (V1rtuousCycle) [reply server/index.js:7]: Nothiing, the line number.
